### DISCUSSION TITLE
Let the "DAX Dependencies" window also show relationships on the selected column

### DISCUSSION THREAD

**Discussion by eybert** [TabularEditor/TabularEditor3#1078] [Ideas]
![image](https://user-images.githubusercontent.com/31990588/142644196-62785163-f596-470c-a3b1-2652b6a67165.png)
This is an example of a foreign key in a fact table where it could be useful to see that it is in fact used in a relationship.

**Comment by mlonsk** [comment]
This feature was delivered a while ago in Tabular Editor 3, apologies for not updating this feature request at the time.
